apps/web/pages/video/[uid].tsx

```diff
@@ -60,11 +80,66 @@ export default function JoinCall(props: JoinCallPageProps) {
       url: meetingUrl,
       ...(typeof meetingPassword === "string" && { token: meetingPassword }),
     });
+    //TODO - add handlers for `transcription-started` and `transcription-ended` @link https://docs.daily.co/reference/rn-daily-js/events/transcription-events
+    /** handling custom-button-click @link  https://docs.daily.co/reference/daily-js/events/meeting-events#custom-button-click */
+    const onCustomButtonClick = async (event?: DailyEventObjectCustomButtonClick | undefined) => {
+      console.log(`calAiCredential ${JSON.stringify(props.calAiCredential)}`);
+      const calAiKeys = await getAppKeysFromSlug("cal-ai");
```

Comment:
You can't do this clientside. - getAppKeysFromSlug is server only.

Reply:
oh, okay. I'll pass it in props. thanks!